docs/pages/api-docs/slider-unstyled.md

```diff
@@ -56,6 +57,36 @@ The `ref` is forwarded to the root element.
 
 Any other props supplied will be provided to the root element (native element).
 
+## CSS
+
+| Rule name | Global class | Description |
+|:-----|:-------------|:------------|
+| <span class="prop-name">root</span> | <span class="prop-name">.root-1</span> | Class name applied to the root element. 
```

Comment:
`.root-1`

Reply:
Good catch! The reason for it is that we don't have name for the `SliderUnstyled` :( (MuiSlider). We have few options of how to solve this 
- generate this name from the component name for the unstyled components -> ComponentNameUnstyled -> MuiComponentName
- bring back the json only for this prop 
- add a static field on the component for it.

I would vote for the first option.

Reply:
Here are the changes - https://github.com/mui-org/material-ui/pull/23569/commits/11e29dfdd5ec8fa7675da49b43b99c08c0d2d97d Let me know if you'd like this solved some other way, but I think this should be fine. Cc @eps1lon you may have an opinion on this too

Reply:
I guess it depends. If we export a `classes` object will all the default global class names, then the problem is solved, we can read it.

Reply:
I think it's the best if the description are kept with the types definition, as they are for the other props. Exporting `classes` object can be complementary in my opinion.